RELEASE.md

```diff
@@ -7,15 +7,17 @@ Please follow the established format:
 -->
 # Next release
 
+# Release 9.1.0
+
 ## Major features and improvements
+- Include expandAllPipelines in initial state. (#1896)
+- Allow Kedro-Viz commands to run from any sub directory within Kedro project, and implement _is_project and _find_kedro_project to tackle kedro project not found issue. (#1871)
```

Comment:
Maybe

```suggestion
- Allow Kedro-Viz commands to run from any sub directory within Kedro project (#1871)
```

?

Reply:
what do you think @rashidakanchwala? Yesterday you suggested to include the other part of the ticket too, which is the `implement _is_project and _find_kedro_project to tackle kedro project not found issue.`, and I also include it in the reminder-panel. Happy to remove it if you don't think we need to mention it